src/content/ModalLayer.tsx

```diff
@@ -94,6 +103,62 @@ function ModalLayer(props: ModalLayerProps) {
           hasTintedBg: false,
         }
       }
+      case ModalPreset.SyncErrors: {
+        const data: SyncScheduleModalData =
+          action.additionalData as SyncScheduleModalData
+
+        const errors = (
+          <ul style={{ fontSize: "1.2em", padding: "10px" }}>
+            {data.syncErrors.map((error, index) => (
+              <li key={index}>
+                {index + 1}. {error}
+                <br />
+              </li>
+            ))}
+          </ul>
+        )
+
+        return {
+          title: "Error Syncing With Saved Schedule",
+          body: errors,
+          closeButtonText: "Close",
+          actionButtonText: "OK",
+          actionHandler: data.onConfirm,
+          cancelHandler: data.onCancel,
+          hasTintedBg: false,
+          actionType: ModalActionType.Normal
+        }
+      }
+      case ModalPreset.SyncInstructions: {
+        const data: SyncScheduleModalData =
+          action.additionalData as SyncScheduleModalData
+
+        return {
+          title: "Sync Saved Schedules Instructions",
+          body: `Please note that you must be on the "View Saved Schedules" page. If you have multiple schedules, click the "add course sections" button on the one you which to add to, otherwise it will add to the first one. You must have all requirements (for example class requires lab and lecture) in your worklist`,
+          closeButtonText: "Close",
+          actionButtonText: "OK",
+          actionHandler: data.onConfirm,
```

Comment:
for this modal and the one below it, there's no actual action to be taken - you pass the same handlers for both the cancelHandler and the actionHandler. 

i'd suggest dropping all of the button-related properties - the modal layer will automatically add an "OK" button that closes the modal. See `Modal.HidePfpInfo` for an example